bin/apisix

```diff
@@ -560,6 +560,21 @@ local function init_etcd(show_output)
     for index, host in ipairs(yaml_conf.etcd.host) do
         local is_success = true
 
+        local token_head = ""
+        if etcd_conf.user and etcd_conf.password then
+            local uri_auth = host .. "/v3/auth/authenticate"
+            local post_json_auth = '{"name": "' .. etcd_conf.user .. '", "password": "' .. etcd_conf.password .. '"}'
+            local cmd_auth = "curl -s " .. uri_auth .. " -X POST -d '" .. post_json_auth
+                        .. "' --connect-timeout " .. timeout
+                        .. " --max-time " .. timeout * 2 .. " --retry 1 2>&1"
+
+            local res_auth = execute_cmd(cmd_auth)
+            local body_auth, _, err_auth = dkjson.decode(res_auth)
+            if not err_auth then
+                token_head = " -H 'Authorization: " .. body_auth.token .. "'"
```

Comment:
Need to show error message and stop the command when failed.

Reply:
Error message shoud be show, but I think it's not necessary to stop when the command failed

Reply:
@jxhecong
We should stop the command when the authorization is failed. Otherwise, we won't init the etcd correctly, just like what happen now.

Reply:
Agree, once the authentication failed, we have to terminate the command.

Reply:
Done